srcpkgs/qtile/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'qtile'
+pkgname=qtile
+version=0.13.0
+revision=1
+build_style=python3-module
+make_install_args="--skip-build"
+hostmakedepends="python3-setuptools python3-cairocffi python3-xcffib"
+makedepends="python3-devel libffi-devel"
+depends="python3-cairocffi python3-xcffib"
```

Comment:
I think a dependency is missing.
When try to run qtile I get
```
$ qtile
Traceback (most recent call last):
  File "/usr/bin/qtile", line 6, in <module>
    from pkg_resources import load_entry_point
ModuleNotFoundError: No module named 'pkg_resources'
```
pkg_resources is in python3-setuptools, when that is installed qtile starts.

Reply:
`python3-setuptools` appended to `depends`.